src/tests/frontend/specs/clear_authorship_colors.js

```diff
@@ -98,4 +99,46 @@ describe('clear authorship colors button', function () {
     await helper.waitForPromise(
         () => chrome$('div.disconnected').attr('class').indexOf('visible') === -1);
   });
+
+  it("clears authorship on single line list item", async function () {
+    this.timeout(1500);
+    const inner$ = helper.padInner$;
+    const chrome$ = helper.padChrome$;
+
+    // override the confirm dialogue functioon
+    helper.padChrome$.window.confirm = () => true;
+
+    // Set some new text
+    const sentText = 'Hello';
+
+    const textElement = inner$('div');
+    inner$('div').parent().children('div').remove();
+
+    // wait until everything is removed then send the string
+    await helper.waitForPromise(() => inner$('div').length === 1);
+    inner$('div').first().sendkeys(sentText);
```

Comment:
This can be simplified:

```suggestion
    await helper.clearPad();
    await helper.edit('Hello');
```